ImguiMapEditor/Services/SettingsRegistry.cpp

```
@@ -21,7 +21,10 @@ bool SettingsRegistry::load() {
   recent_locations_->loadFromConfig(*config_service_);
 
   view_settings_.loadFromConfig(*config_service_);
-  selection_settings_.loadFromConfig(*config_service_);
+  // SelectionSettings serialization (was in Domain/SelectionSettings.cpp — moved here to break Domain→Services dependency)
+  selection_settings_.floor_scope = static_cast<Domain::SelectionFloorScope>(
```

Cppcheck (2.20.0)

[style] The function 'getName' is never used.

(unusedFunction)